german/ruderboot.md

```diff
@@ -1,4 +1,4 @@
-Sie wachen von der Sonne geblendet auf und stellen ziemlich benommen fest, dass Sie allein 
```

Comment:
Hi Caroline,

thanks for the great course.
Yes the change was intentional. "allein" means "alone". In the german
version someone wakes up alone in a boat on a big lake. I want some more
options for the story to continue, so in my version, he isn't waking up
alone anymore, (but with two dead bodies beside him, which is kinda a form
of alone too :§ )

best regards
Lukas

On 13 July 2015 at 19:47, Caroline Buckey <notifications@github.com> wrote:

> In german/ruderboot.md
> <https://github.com/udacity/create-your-own-adventure/pull/1852#discussion_r34490180>
> :
>
> > @@ -1,4 +1,4 @@
> > -Sie wachen von der Sonne geblendet auf und stellen ziemlich benommen fest, dass Sie allein
>
> Was deleting the word "allein" intentional? (I don't speak German, so I
> can't tell whether the change makes sense.) The other two "modifications"
> look like they didn't actually change anything, so if this one was
> intentional I'll be happy to merge this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/udacity/create-your-own-adventure/pull/1852/files#r34490180>
> .
>